crates/biome_html_analyze/tests/specs/nursery/useControlLabel/html/invalid.html

```
@@ -0,0 +1,45 @@
+<!-- should generate diagnostics -->
+
+<button></button>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)